### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by gtekle**
## What this PR does:
 - Created vet_clinic database
 - Create animals table
 - Inserted initial data into animals table using SQL INSERT statements
 - Wrote queries on animals table using SQL SELECT statement based on various criteria.
 - Updated README file

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name like '%mon';
+
+-- SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (mmsesay) [inline queries.sql]: - [ ] This query is good. But what if an animal was born on the 21st March 2019? Currently, your query doesn't cover those animals. You only cover animals born on the 1st  day and 1st month of 2016 and 2019.
Please update this query to cover animals born in 2019.

See example:
```sql
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @gtekle, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (gtekle) [reply queries.sql]: Hi @mmsesay
This was a good catch! :+1: 
Thank you very much for the feedback. :bow:

**Review by t-yanick — APPROVED** [review]
Hi @gtekle, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (mmsesay) [reply queries.sql]: Good job working on the changes. 
The pleasure is all mine @gtekle.

**Comment by gtekle** [comment]
Hi @t-yanick,
Thanks :+1:

**Pull request merged**